### PR TITLE
Konrad branch

### PR DESCRIPTION
**Pull request by koporski**
Please review this, @casey-collab

### CODE REVIEW

File: Polish/ubranie/kieszen/gwizdek/dmuchnij-w-gwizdek.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Dmuchasz w gwizdek. Niespodziewanie przenosisz się do innego wymiaru.
```
Comment (casey-collab) [inline Polish/ubranie/kieszen/gwizdek/dmuchnij-w-gwizdek.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: Polish/ubranie/kieszen/gwizdek/wyrzuc-gwizdek.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Wyrzucasz gwizdek. Niespodziewanie trafiasz nim w swoją matkę.
```
Comment (casey-collab) [inline Polish/ubranie/kieszen/gwizdek/wyrzuc-gwizdek.md]: This file has also been added to a directory that already contained a page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**